src/zxcvbn_password/validators.py

```diff
@@ -20,7 +21,8 @@ class ZXCVBNValidator(object):
     DEFAULT_USER_ATTRIBUTES = ('username', 'first_name', 'last_name', 'email')
 
     def __init__(self, min_score=DEFAULT_MIN_SCORE,
-                 user_attributes=DEFAULT_USER_ATTRIBUTES):
+                 user_attributes=DEFAULT_USER_ATTRIBUTES,
+                 frequency_lists={}):
```

Comment:
Prefer to use `frequency_lists=None` and a condition in the method body:

```python
if not frequency_lists:
    frequency_lists = {}
```

Using mutables as defaults in functions parameters is a code smell 🙂 (each time you instantiate ZXCVBNValidator, the same empty dict reference is used, so modifying it in instance A would also modify it in instance B).

Reply:
Thanks! I changed it to

    self.frequency_lists = frequency_lists or {}

Hope, that's ok.